mithril-aggregator/src/commands/mod.rs

```diff
@@ -45,6 +45,7 @@ impl MainCommand {
             Self::Serve(cmd) => cmd.execute(root_logger, config_builder).await,
             Self::Tools(cmd) => cmd.execute(root_logger, config_builder).await,
             Self::GenerateDoc(cmd) => {
+                println!("Please note: the documentation generated is not able to indicate the environment variables used by the commands.");
```

Comment:
Is this the correct location for this println ? Should it be in `mithril-doc` instead ?

Reply:
Done